src/jit/lsra.cpp

```diff
@@ -12821,12 +12863,19 @@ void LinearScan::verifyFinalAllocation()
                                 assert(!compiler->lvaTable[compiler->lvaTrackedToVarNum[varIndex]].lvLRACandidate);
                                 continue;
                             }
-                            regNumber regNum                  = getVarReg(inVarToRegMap, varIndex);
-                            interval                          = getIntervalForLocalVar(varIndex);
-                            interval->physReg                 = regNum;
-                            interval->assignedReg             = &(physRegs[regNum]);
-                            interval->isActive                = true;
-                            physRegs[regNum].assignedInterval = interval;
+                            regNumber regNum      = getVarReg(inVarToRegMap, varIndex);
+                            interval              = getIntervalForLocalVar(varIndex);
+                            interval->physReg     = regNum;
+                            interval->assignedReg = &(physRegs[regNum]);
+                            interval->isActive    = true;
+#ifdef _TARGET_ARM_
+                            if (physRegs[regNum].registerType == TYP_DOUBLE)
```

Comment:
None of the `RegRecord`s are `TYP_DOUBLE`; they are all set as `TYP_FLOAT`, see https://github.com/dotnet/coreclr/blob/master/src/jit/lsra.h#L222

Reply:
That's my mistake, I got it. Thank you!